test/conversions.js

```diff
@@ -582,6 +582,54 @@ const tests = {
 				}
 			]
 		},
+		{
+			name: "HCT",
+			data: {
+				toSpace: "hct",
+			},
+			tests: [
+				{
+					name: "sRGB white to CAM16 JMh",
```

Comment:
to CAM16 JMh or to HCT?

Reply:
Sorry, missed this. I copied the CAM16 JMh tests tests and altered them for HCT. I forgot to change the names though.

Reply:
Fixed in latest commit